.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
     "stylelint.packageManager": "yarn",
-    "cSpell.enabled": true
+    "cSpell.enabled": true,
+    "cSpell.words": [
+        "prechecks"
```

Comment:
Why is this change required?

Reply:
I received multiple cSpell errors regarding unrecognized words while attempting to commit. I assumed that it was necessary to include them in this file.

Reply:
I have just realized that this is a file for the editor. It seems that I may have created it unconsciously. I'll revert this change.